src/NationalInstruments.Analyzers/Correctness/ApprovedNamespaceCodeFixProvider.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using NationalInstruments.Analyzers.Properties;
+
+namespace NationalInstruments.Tools.Analyzers.Namespaces
```

Comment:
Update the namespace to match the analyzer

Reply:
Fixed it.